modules/gitops-release-notes-1-4-0.adoc

```diff
@@ -0,0 +1,52 @@
+
+* As an administrative user, when you give Argo CD access to a namespace by using the `argocd.argoproj.io/managed-by` label, it assumes namespace-admin privileges. These privileges are an issue for administrators who provide namespaces to non-administrators, such as development teams, because the privileges enable non-administrators to modify objects such as network policies.
++
+With this update, administrators can configure a common cluster role for all the managed namespaces. In role bindings for the Argo CD application controller, the Operator refers to the `CONTROLLER_CLUSTER_ROLE` environment variable. In role bindings for the Argo CD server, the Operator refers to the `SERVER_CLUSTER_ROLE` environment variable. If these environment variables contain custom roles, the Operator does not create the default admin role. Instead, it uses the existing custom role for all managed namespaces.
+link:https://issues.redhat.com/browse/GITOPS-1290[GITOPS-1290]
+
+* With this update, the Environment page in the {product-title} Developer Console displays a broken heart icon to indicate degraded resources (excluding Progressing, Missing, and Unknown) and a yellow yield sign icon to indicate out-of-sync resources. link:https://issues.redhat.com/browse/GITOPS-1307[GITOPS-1307]
+
+[id="fixed-issues-1-4-0_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, when the Route to the {gitops-title} Application Manager (kam) was accessed without specifying a path in the URL, a default page without any helpful information was displayed to the user. The current update fixes this issue so that the default page displays download links for kam. link:https://issues.redhat.com/browse/GITOPS-923[GITOPS-923]
+
+* Before this update, setting a resource quota in the namespace of the Argo CD custom resource might cause the setup of the Red Hat SSO (RH SSO) instance to fail. This update fixes this issue by setting a minimum resource request for the RH SSO deployment pods. link:https://issues.redhat.com/browse/GITOPS-1297[GITOPS-1297]
+
+* Before this update, if you changed the log level for the `argocd-repo-server` workload, the Operator didn't reconcile this setting. The workaround was to delete the deployment resource so that the Operator recreated it with the new log level. With this update, the log level is correctly reconciled for existing `argocd-repo-server` workloads. link:https://issues.redhat.com/browse/GITOPS-1387[GITOPS-1387]
```

Comment:
Remove the contraction: `...the Operator did not reconcile this setting.` 

I don't think anything is wrong with contractions. However, you don't use contractions throughout the RN, so for consistency, avoid the few instances when you do.

Reply:
IBM SG: "Use simple, commonly understood contractions."
Thanks. I updated the document to use contractions consistently.